examples/api-routes-apollo-server-and-client/lib/with-apollo-client.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import initApollo from './init-apollo'
+import Head from 'next/head'
+import { getDataFromTree } from '@apollo/react-ssr'
```

Comment:
Lets remove `@apollo/react-ssr` from the client bundle by using an inline import:

https://github.com/zeit/next.js/blob/canary/examples/with-apollo/lib/apollo.js#L70-L78